Dockerfile

```diff
@@ -35,8 +35,10 @@ ONBUILD ARG LOCAL_CUSTOM_DIR=./custom
 ONBUILD COPY $LOCAL_CUSTOM_DIR /opt/odoo/custom
 # https://docs.python.org/2.7/library/logging.html#levels
 ONBUILD ARG LOG_LEVEL=INFO
+ONBUILD RUN ln -s /opt/odoo/custom/ssh ~root/.ssh \
+            && chmod -Rc 0600 /opt/odoo/custom/ssh/* \
+            && chmod -c 0700 /opt/odoo/custom/ssh/
```

Comment:
- move this to common/build.d/80-permissions.
- do not use octal permissions, they are less readable than using letters.
- a single `chmod -Rc u=rwX,go= custom/ssh` would do the same
- the `ln -s` part should be moved to the base image (outside an `ONBUILD`) because symlink creations don't need an existing target to success, so we drop one layer from the onbuild and move it to base image. (This was my fault, but now that you're fixing it... 😇 )